de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/fresh_PriceList_Details_Template_Report.sql

```diff
@@ -28,25 +28,25 @@ AS
-       CONCAT(bp_value, '_', bp_name, '_', case when prlv.isactive = 'Y' then prlv.validfrom::date else null end, '.xls') as reportfilename
+SELECT plc.value                                                                                                                   AS prodvalue,
+       plc.customerproductnumber                                                                                                   as customerproductnumber,
+       plc.productcategory                                                                                                         as productcategory,
+       plc.productname                                                                                                             as productname,
+       plc.attributes                                                                                                              as attributes,
+       coalesce(hupip.name, hupiv.description, plc.uomsymbol)                                                                      as itemproductname,
+       NULL::numeric                                                                                                               as qty,
+       plc.uomsymbol                                                                                                               as uomsymbol,
+       round(plc.pricestd, cur.stdprecision)                                                                                       as pricestd,
+       plc.M_ProductPrice_ID                                                                                                       as m_productprice_id,
+       p_c_bpartner_id                                                                                                             as c_bpartner_id,
+       plc.M_HU_PI_Item_Product_ID                                                                                                 as m_hu_pi_item_product_id,
+       case when plc.m_hu_pi_item_product_id is not null then 'COLI' else plc.uom_x12de355 end                                   as uom_x12de355,
+       p_c_bpartner_location_id                                                                                                    as c_bpartner_location_id,
+       plc.qtycuspertu                                                                                                             as qtycuspertu,
+       plc.m_product_id                                                                                                            as m_product_id,
+       plc.BP_Value                                                                                                                as bp_value,
+       plc.BP_Name                                                                                                                 as bp_name,
+       CONCAT(bp_value, ''_'', bp_name, ''_'', case when prlv.isactive = ''Y'' then prlv.validfrom::date else null end, ''.xls '') as reportfilename
```

Comment:
i think the double `''` are wrong in here. i think it's fine in the migration file, but future devs might use *this* file as their base to extend the feature